tests/plugins/uploadimage/uploadimage.js

```diff
@@ -550,6 +552,30 @@
 				} );
 			} );
 
+			wait();
+		},
+
+		'test uploads generate unique names (#1213)': function() {
```

Comment:
Test is always failing:

> TypeError: Attempted to wrap create which is already wrapped

Reply:
Ooops, TBH don't know how I missed that.